pkg/mattermost/mattermost_v1beta.go

```diff
@@ -252,6 +252,16 @@ func GenerateDeploymentV1Beta(mattermost *mmv1beta.Mattermost, db DatabaseConfig
 
 	liveness, readiness := setProbes(mattermost.Spec.Probes.LivenessProbe, mattermost.Spec.Probes.ReadinessProbe)
 
+	var containerSecurityContext *corev1.SecurityContext
+	if mattermost.Spec.PodTemplate != nil && mattermost.Spec.PodTemplate.ContainerSecurityContext != nil {
+		containerSecurityContext = mattermost.Spec.PodTemplate.ContainerSecurityContext
+	}
+
+	var podSecurityContext *corev1.PodSecurityContext
+	if mattermost.Spec.PodTemplate != nil && mattermost.Spec.PodTemplate.SecurityContext != nil {
+		podSecurityContext = mattermost.Spec.PodTemplate.SecurityContext
+	}
```

Comment:
Since both security context are `nil` by default, we can perhaps simplify this a bit:
```suggestion
	var containerSecurityContext *corev1.SecurityContext
	var podSecurityContext *corev1.PodSecurityContext
	if mattermost.Spec.PodTemplate != nil {
		containerSecurityContext = mattermost.Spec.PodTemplate.ContainerSecurityContext
		podSecurityContext = mattermost.Spec.PodTemplate.SecurityContext
	}
```

Reply:
Applied, thanks!